web/cobrands/bromley/assets.js

```diff
@@ -67,6 +67,17 @@ var highways_stylemap = new OpenLayers.StyleMap({
     })
 });
 
+var bromley_to_tfl = {
```

Comment:
Just to be clear, these are Bromley categories where we want to send the report only to TfL? (assuming it's on a red route)

Reply:
Correct.